go.mod

```diff
@@ -3,23 +3,23 @@ module github.com/crescent-network/crescent/v3
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.9
-	github.com/cosmos/ibc-go/v2 v2.2.0
+	github.com/cosmos/cosmos-sdk v0.45.10
```

Comment:
Do we need to bump Cosmos SDK to `v0.45.10` as well in this patch? The `replace` directive in this file specifies `github.com/cosmos/cosmos-sdk => github.com/crescent-network/cosmos-sdk v1.1.3-sdk-0.45.9`

Reply:
Will upgrade Cosmos SDK from v0.45.9 to v0.45.10 now and apply it to this PR.

Reply:
Applied.